controller/src/test/java/io/pravega/controller/server/bucket/PravegaTablesStoreBucketServiceTest.java

```diff
@@ -67,4 +79,53 @@ BucketStore createBucketStore(int bucketCount) {
                 BucketStore.ServiceType.WatermarkingService, bucketCount);
         return StreamStoreFactory.createZKBucketStore(map, zkClient, executor);
     }
+
+    @Override
+    protected void addEntryToZkCluster(Host host)  {
```

Comment:
Again, repeated code: please, try to reuse this kind of functions to do not write them over and over again.

Reply:
Removed repeated code.